src/components/tagsInput/index.js

```diff
@@ -310,16 +337,21 @@ export default class TagsInput extends BaseComponent {
   }
 
   render() {
-    const {disableTagRemoval, containerStyle, hideUnderline} = this.getThemeProps();
+    const {disableTagRemoval, containerStyle, hideUnderline, validationErrorMessage} = this.getThemeProps();
     const tagRenderFn = disableTagRemoval ? this.renderTag : this.renderTagWrapper;
-    const {tags} = this.state;
+    const {tags, tagIndexToRemove} = this.state;
 
     return (
       <View style={[!hideUnderline && styles.withUnderline, containerStyle]}>
         <View style={styles.tagsList}>
           {_.map(tags, tagRenderFn)}
           {this.renderTextInput()}
         </View>
+        <View>
+          <Text style={[validationErrorMessage && tagIndexToRemove ? styles.inValidMarkedTagLabel : styles.inValidTagLabel]}>
```

Comment:
regarding the the style 
we usually have a base style for the element and we only condition the change. What does it mean? 
In this case I would have a base style for the error message, for example: `styles.errorMessage`, and it will contain the base props, in our case it's 
```
errorMessage: {
  ...Typography.text80,
  color: Colors.red30
}
```
Now, I will add a style for the "markedTagLabel" case, and it will contain only the difference 
```
errorMessageWhileMarked: {
  color: Colors.red10
}
```
And the way we assign the style is like this
```
<Text style={[styles.errorMessage, tagIndexToRemove && styles. errorMessageWhileMarked]}
```
The reason for that is so we don't have duplicates of styles (like the Typography.text80)
if the design changes in the future, we change it only at one place and not several places.

Reply:
done , did it also for the invalid tag & removeIcon styles